common.sh

```diff
@@ -4,7 +4,7 @@ set -eu
 
 # Default values for 'latest' tag
 latestMavenVersion='3.9.6'
-latest='21'
+latest='22'
```

Comment:
eclipse-temurin latest is still pointing to 21
https://github.com/docker-library/official-images/blob/master/library/eclipse-temurin#L370